specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/examples/LogAnalyticsRequestRateByInterval.json

```diff
@@ -0,0 +1,34 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "location": "westus",
+    "api-version": "2017-12-01",
+    "parameters": {
+      "properties": {
```

Comment:
Please extract everything out of both "RequestRateByIntervalInput" and "properties". The "parameters" section should contain the parameters in this format:
```json
"<parameter-name>": <actual payload>
```

In this case, the actual payload is 
```json
{
  "intervalLength": "FiveMins",
  "blobContainerSasUri": "https://somesasuri",
  "fromTime": "2018-01-21T01:54:06.862601Z",
  "toTime": "2018-01-23T01:54:06.862601Z",
  "groupByResourceName": true
 }
```

You can see another example in compute here: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/examples/CreateAScaleSetFromACustomImage.json#L8

Reply:
Same for the other example.